projects/igniteui-angular/src/lib/core/styles/themes/schemas/light/_expansion-panel.scss

```diff
@@ -65,3 +65,30 @@ $_fluent-expansion-panel: extend($_light-expansion-panel);
 /// @requires {Map} $_light-expansion-panel
 $_bootstrap-expansion-panel: extend($_light-expansion-panel, $_bootstrap-shape-expansion-panel);
 
+// Generates an indigo expansion panel schema.
+/// @type {Map}
+/// @requires {function} extend
+/// @requires {Map} $_light-expansion-panel
+/// @requires {Map} $_indigo-shape-expansion-panel
+$_indigo-expansion-panel: extend(
+    $_light-expansion-panel,
+    $_indigo-shape-expansion-panel,
+    (
+        header-title-color: (
+            igx-color: ('grays', 900)
+        ),
+
+        header-description-color: (
+            igx-color: ('grays', 700)
+        ),
+
+        header-focus-background: (
+            igx-color: ('primary', 500),
+            rgba: .15
+        ),
+
+        disabled-color: (
+            igx-color: ('grays', 400)
+        )
+    )
+);
```

Comment:
The value for "header-icon-color" is missing ('grays', 600).
You should also provide documentation for these properties.